hasura.planx.uk/migrations/1689681979734_split_out_breadcrumbs_and_passport/up.sql

```diff
@@ -0,0 +1,7 @@
+ALTER TABLE "public"."sessions" RENAME COLUMN "data" TO "breadcrumbs";
+
+ALTER TABLE "public"."sessions" ADD COLUMN "passport" jsonb
+ NULL;
```

Comment:
Question: if the passport can always be derived from the breadcrumbs, what value do you think we get from storing it directly? In moving towards a new data model I've been curious if we can exclusively store breadcrumbs and flow graphs - and have shared methods (eg a new planx-core iteration of `computePassport()`?) to derive all other commonly used data formats? 

Just curious around your thoughts on this! Happy for this to merge as proposed here regardless.

Reply:
Sounds good, thanks for sense checking this. If porting over proves tricky, very happy with a shared short versus long term goal here though ! Agree it would be easy enough to drop the column in the future if simplest to get a basic working example, then move `computePassport` later.